resources/js/sfx.js

```diff
@@ -0,0 +1,214 @@
+		this.create_sfx('Potion refill/Magic drain', 0x03, 0x1D, 0x20A6, []),
+		this.create_sfx('Flapping (Duck/Cucco swarm/Ganon bats/Keese/Raven/Vulture)', 0x03, 0x1E, 0x2091, []),
+		this.create_sfx('Link falling', 0x03, 0x1F, 0x204B, []), this.create_sfx('Menu/Text cursor moved', 0x03, 0x20, 0x276C, []),
+		this.create_sfx('Damage to boss', 0x03, 0x21, 0x27E2, []), this.create_sfx('Boss dying/Deleting file', 0x03, 0x22, 0x26CF, []),
+		this.create_sfx('Spin attack/Medallion swoosh', 0x03, 0x23, 0x2001, [0x39]),
+		this.create_sfx('OW map perspective change', 0x03, 0x24, 0x2043, []),,
+		this.create_sfx('Pressure switch', 0x03, 0x25, 0x1E9D, []),,
+		this.create_sfx('Lightning/Game over/Laser/Ganon bat/Trinexx lunge', 0x03, 0x26, 0x1E7B, []),
+		this.create_sfx('Agahnim charge', 0x03, 0x27, 0x1E40, []), this.create_sfx('Agahnim/Ganon teleport', 0x03, 0x28, 0x26F7, []),
+		this.create_sfx('Agahnim shot', 0x03, 0x29, 0x1E21, []),,
+		this.create_sfx('Somaria/Byrna/Ether spell/Helma fire ball', 0x03, 0x2A, 0x1E12, []),
+		this.create_sfx('Electrocution', 0x03, 0x2B, 0x1DF3, []), this.create_sfx('Bees', 0x03, 0x2C, 0x1DC0, []),
+		this.create_sfx('Milestone, also via text', 0x03, 0x2D, 0x1DA9, [0x37]),,
+		this.create_sfx('Collecting heart container', 0x03, 0x2E, 0x1D5D, [0x35, 0x34]),,
+		this.create_sfx('Collecting absorbable key', 0x03, 0x2F, 0x1D80, [0x33]),,
+		this.create_sfx('Byrna spark/Item plop/Magic bat zap/Blob emerge', 0x03, 0x30, 0x1B53, []),
+		this.create_sfx('Sprite falling/Moldorm shuffle', 0x03, 0x31, 0x1ACA, []),
+		this.create_sfx('Bumper boing/Somaria punt/Blob transmutation/Sprite boings', 0x03, 0x32, 0x1A78, []),
+		this.create_sfx('Jingle (paired $2F→$33)', 0x03, 0x33, 0x1D93, [], true),
+		this.create_sfx('Depressing jingle (paired $2E→$35→$34)', 0x03, 0x34, 0x1D66, [], true),
+		this.create_sfx('Ugly jingle (paired $2E→$35→$34)', 0x03, 0x35, 0x1D73, [], true),
+		this.create_sfx('Wizzrobe shot/Helma fireball split/Mothula beam/Blue balls', 0x03, 0x36, 0x1AA7, []),
+		this.create_sfx('Dinky jingle (paired $2D→$37)', 0x03, 0x37, 0x1DB4, [], true),
+		this.create_sfx('Apathetic jingle (paired $1A→$38)', 0x03, 0x38, 0x1E93, [], true),
+		this.create_sfx('Quiet swish (paired $23→$39)', 0x03, 0x39, 0x2017, [], true),
+		this.create_sfx('Defective jingle (paired $1B→$3A)', 0x03, 0x3A, 0x20C0, [], true),
+		this.create_sfx('Petulant jingle (paired $10→$3B)', 0x03, 0x3B, 0x2176, [], true),
+		this.create_sfx('Triumphant jingle (paired $0F→$3C→$3D→$3E→$3F)', 0x03, 0x3C, 0x248A, [], true),
+		this.create_sfx('Less triumphant jingle ($0F→$3C→$3D→$3E→$3F)', 0x03, 0x3D, 0x2494, [], true),
+		this.create_sfx('"You tried, I guess" jingle (paired $0F→$3C→$3D→$3E→$3F)', 0x03, 0x3E, 0x249E, [], true),
+		this.create_sfx('"You didn\'t really try" jingle (paired $0F→$3C→$3D→$3E→$3F)', 0x03, 0x3F, 0x2480, [], true)
+	];
+  }
+
+  shuffle_sfx_data() {
```

Comment:
Non-blocking: It would be nice to have a doc block explaining why we are only shuffling sfx sets `2` and `3`

Reply:
I stuck this in the code, but repeating here for knowledge:
The first set is mostly sounds that don't lend well to being played randomly due to specially programmed looping in set 1. (e.g. different types of Rain, rumbling, etc)